cli/command/install/tool.go

```diff
@@ -0,0 +1,80 @@
+package install
+
+import (
+	"github.com/fatih/color"
+	"github.com/opencurve/curveadm/cli/cli"
+	comm "github.com/opencurve/curveadm/internal/common"
+	"github.com/opencurve/curveadm/internal/configure/topology"
+	"github.com/opencurve/curveadm/internal/playbook"
+	cliutil "github.com/opencurve/curveadm/internal/utils"
+	"github.com/spf13/cobra"
+)
+
+var (
+	INSTALL_TOOL_PLAYBOOK_STEPS = []int{
+		playbook.INSTALL_TOOL,
+	}
+)
+
+type installOptions struct {
+	host string
+}
+
+func NewInstallToolCommand(curveadm *cli.CurveAdm) *cobra.Command {
+	var options installOptions
+
+	cmd := &cobra.Command{
+		Use:   "tool [OPTIONS]",
+		Short: "Install tool v2 on the specified host",
+		Args:  cliutil.NoArgs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runInstallTool(curveadm, options)
+		},
+		DisableFlagsInUseLine: true,
+	}
+
+	flags := cmd.Flags()
+	flags.StringVar(&options.host, "host", "localhost", "Specify target host")
+
+	return cmd
+}
+
+func genInstallToolPlaybook(curveadm *cli.CurveAdm,
+	dcs []*topology.DeployConfig,
+	options installOptions,
+) (*playbook.Playbook, error) {
+	configs := curveadm.FilterDeployConfigByRole(dcs, topology.ROLE_MDS)[:1]
```

Comment:
The host need to filter also becasue we need to copy from container that run on the host.